quickstartproject/production.py

```diff
@@ -20,3 +20,5 @@
 
 STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'  
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+
+Debug = False
```

Comment:
I guess it must not matter, but can you re-capitalize for consistency?

```suggestion
DEBUG = False
```